tests/test_cli.sh

```diff
@@ -1,14 +1,42 @@
+#!/bin/zsh
+
 RUNNER_TEMP=~/test_temp
-LOG_LEVEL=debug
+LOG_LEVEL=info
+FORMAT=$1
+if [[ "$FORMAT" -ne "zarr" && "$FORMAT" -ne "asdf" ]]; then
+       echo "Missing FORMAT argument (zarr or asdf)"
+       exit 1
+fi
+echo "FORMAT is $FORMAT"
+LOGFILE="$HOME/logs/log_${FORMAT}_$(date -j +'%Y%m%d_%H%M%S').txt"
+STATIONS=ARV,BAK
+START=2019-02-01T00:00:00
+END=2019-02-01T01:00:00
+INC_HOURS=1
+
```

Comment:
Results for this "bigger test":
![image](https://github.com/mdenolle/NoisePy/assets/10456546/c19cca54-7989-4a6b-a23f-2cdf4fdf2d0f)
This is running on my Mac, so local SSD I/O. All times in seconds.
TL; DR: Zarr is slower for writing, but faster for reading. But also note that there is not a huge difference in the overall time. However, this may change with: a) parallelization of stacking and b) cloud storage